lib/openstudio/alfalfa/creator.rb

```diff
@@ -150,13 +153,111 @@ def resolve_template(mapping)
 
       def add_relationship_info(obj, relationships, info)
         relationships.each do |relationship|
-          ref = obj.send(relationship['openstudio_method'])
-          break unless !ref.empty?
+          ref = []
+          relationship['openstudio_method'].each do |method|
+            ref = obj.send(method)
+            break unless !ref.is_initialized
+          end
+          break unless ref.is_initialized
```

Comment:
there's gotta be a less janky way to do this

Reply:
it works though? you can take a crack :)